build.gradle

```diff
@@ -53,7 +53,8 @@ dependencies {
     def kotlinVersion = '1.3.21'
 
     compile gradleApi()
-    compile localGroovy()
+    // compile localGroovy()
```

Comment:
Usually this is so I can test with different cradle versions, knowing their respective groovy version will work.